File: src/main/java/me/basiqueevangelist/datapackify/potionrecipes/PotionRecipes.java
package me.basiqueevangelist.datapackify.potionrecipes;

import net.fabricmc.fabric.api.resource.ResourceManagerHelper;
import net.minecraft.resource.ResourceType;

public final class PotionRecipes {
    private PotionRecipes() {

    }

    public static void init() {
        ResourceManagerHelper.get(ResourceType.SERVER_DATA).registerReloadListener(new BrewingRecipeManager());
    }
}

File: src/main/kotlin/ml/porez/datapackify/potionrecipes/PotionRecipes.kt
package ml.porez.datapackify.potionrecipes

import net.fabricmc.fabric.api.resource.ResourceManagerHelper
import net.minecraft.resource.ResourceType

object PotionRecipes {
    private val MANAGER = BrewingRecipeManager()

    fun init() {
        ResourceManagerHelper.get(ResourceType.SERVER_DATA).registerReloadListener(MANAGER)
    }
}